app/forms/openid_connect_authorize_form.rb

```diff
@@ -275,6 +310,19 @@ def scopes
     OpenidConnectAttributeScoper::VALID_IAL1_SCOPES
   end
 
+  def parsed_vector_of_trust
+    return @parsed_vector_of_trust if defined?(@parsed_vector_of_trust)
+    return @parsed_vector_of_trust = nil if vtr.blank?
+
+    @parsed_vector_of_trust = begin
+      if vtr.is_a?(Array) && !vtr.empty?
+        Vot::Parser.new(vector_of_trust: vtr.first).parse
+      end
+    rescue Vot::Parser::ParseException
+      nil
```

Comment:
would we want to catch this exception and add an error for it? or no

Reply:
The `validate_vtr` param has kind of a roundabout way of doing it. It looks like see if the param is present and this is nil.

I would like to find a more direct and elegant way of doing it. The trick with this getting reliably re-evaluated with `valid?` calls could get messy.